src/components/pages/Home.tsx

```diff
@@ -88,18 +101,9 @@ export default function HomePage(): ReactElement {
         <Bookmarks />
       </section>
 
-      <SectionQueryResult
-        title="Highest Liquidity"
-        query={queryHighest}
-        action={
-          <Button
-            style="text"
-            to="/search?priceType=pool&sort=liquidity&sortOrder=desc"
-          >
-            Data sets and algorithms with pool →
-          </Button>
-        }
-      />
+      {searchDids && (
```

Comment:
the section needs to render at all times from the beginning to prevent bigger layout shifts and set user expectations by having the section title present at all times. If the component breaks because some data is missing it should be solved in the component, so as to have it use its loading state longer